core/src/test/scala/unit/kafka/integration/KafkaServerTestHarness.scala

```diff
@@ -248,9 +248,11 @@ abstract class KafkaServerTestHarness extends QuorumTestHarness {
     index
   }
 
-  def killBroker(index: Int): Unit = {
+  def killBroker(index: Int): Unit = killBroker(index, TimeUnit.SECONDS.toMinutes(5))
```

Comment:
Thanks for the update, can we unify these 2 methods into one: 
`def killBroker(index: Int, timeoutMs: Long = TimeUnit.SECONDS.toMinutes(5)): Unit = {...}`

Reply:
Thanks for giving it another look. I didn't initially do this because the method is also called from Java, where default parameters are not supported. However, it's just one occurrence, so I've updated the caller in that case. I also noticed that `toMinutes` here should be `toMillis` instead.